crates/swc_ecma_transforms_module/src/system_js.rs

```diff
@@ -21,6 +22,9 @@ pub struct Config {
 
     #[serde(default)]
```

Comment:
I think you should remove this

Reply:
@kdy1 - I didn't add this attribute.  Did you mean the inner Config attribute below it or were you hoping I would clean this up?  Also, I'm assuming you would want this changed across all Module Shapes or just this one?

Reply:
I mean the whole `resolve_fully: bool`. It's now duplicated as you added a field below.

Reply:
Oh I see.  Thanks for clarifying. Okay, I've updated that for the ESModuleConfig and the SystemJS Config.